etc/nsjail/execute.cfg

```diff
@@ -0,0 +1,137 @@
+name: "compiler explorer compilation/tool sandbox"
+
+mode: ONCE
+hostname: "ce"
+
+time_limit: 0
+max_cpus: 1
+
+log_level: INFO
+
+rlimit_as_type: INF
+rlimit_cpu_type: SOFT
+rlimit_fsize: 1024 # 1GB
+rlimit_nofile: 20
```

Comment:
our firejail config does not limit file handles for execution, and 20 file handles seems... optimistic.
I am willing to bet many compilers open significantly more files at once.

Reply:
Yup will bump.